gprofiler/spark/sampler.py

```diff
@@ -47,204 +39,19 @@ def __init__(
         self._sample_period = sample_period
         # not the same instance as GProfiler._stop_event
         self._stop_event = Event()
-        self._spark_sampler: Optional[SparkCollector] = None
+        self._spark_sampler: Optional[BigDataSampler] = None
```

Comment:
```suggestion
        self._big_data_sampler: Optional[BigDataSampler] = None
```

Reply:
Match the name?

Reply:
https://github.com/Granulate/gprofiler/pull/749/files#diff-18dce65a9628cdb97bacc94da9b0401f8686d3326c5a71c91a23eb0bf4cbb919R50